.github/workflows/platform-install-advanced.yml

```diff
@@ -8,9 +8,6 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
```

Comment:
```suggestion
    name: Platform Advanced Installation
on:
  push:
    branches: [ master ]
  repository_dispatch:
    types: [qa]
  workflow_dispatch:
jobs:
  build:
    runs-on: ubuntu-latest
    services:
      firefox:
        image: selenium/node-firefox:3.141.59-gold
        env:
          HUB_HOST: hub
          HUB_PORT: 4444
    steps:
      - name: Set up JDK 1.8
        uses: actions/setup-java@v1
        with:
          java-version: 1.8
      - name: Checkout openmrs-core
        uses: actions/checkout@master
        with:
          repository: openmrs/openmrs-core
      - name: build platform
        run: |
          mvn clean install -DskipTests=true
          cp webapp/target/openmrs.war ../
      - name: Checkout qaframework
        uses: actions/checkout@master
        with:
          repository: ${{github.repository}}
      - name: run db and web containers
        run: |
          cp ../openmrs.war docker/
          cd docker
          docker-compose -f docker-compose-platform-install.yml up -d
      - name: wait for openmrs instance to start
        run: while ! nc -z localhost 8080 </dev/null; do sleep 1; done
      - name: Run qaframework on selenium
        run: |
          echo "cucumber.publish.enabled=true" > qaframework-bdd-tests/src/test/resources/cucumber.properties
          mvn clean install -DskipTests=true
          cp -f qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test-local.properties qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test.properties
          cd qaframework-bdd-tests
          npm run advancedCoreInstall
          curl -sX POST -F messages=@target/cucumber.ndjson https://studio.cucumber.io/cucumber_project/results -H "project-access-token: ${{secrets.CUCUMBER_IO_TOKEN}}" -H "provider: github" -H "repo: ${{github.repository}}" -H "branch: master" -H "revision: ${{github.sha}}"

```

Reply:
Thanks @sherrif10 for the suggestion:

I noticed for platform tests doesn't need to set the working directory as `qaframework-bdd-tests` because they depend on other module when installing dependencies. 

What we simply have to do is checkout the directory when building qaframework as suggested in the pull request. With the changes in this PR will make things right.